packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping.md

```diff
@@ -259,7 +258,22 @@ For icons, if multiple values map to the same icon, it is possible that sorting
 
 ### Sizing
 
-`nimble-table-column-icon` will support only a fixed width. We will introduce a new mixin for fixed-width support that exposes a `pixel-width` property. The default value will be the minimum supported by the table, which is still significantly larger than the width of an icon.
+`nimble-table-column-icon` will be a fixed pixel size (32px) and will not be resizable. The 32px fixed size allows room from a single icon along with left and right cell padding of 8px each.
+
+This has the following implications:
+
+-   The grouping indicator and sorting indicator will always be hidden on the icon column.
+-   A client is expected to only place an icon as the header content of an icon column.
```

Comment:
Idle thoughts for implementation / future docs, no action needed now:
1. Some SLE grids like Systems currently include an icon column with no header content so we'll either want to support that (I assume it works already) or suggest they start using an icon header
2. We'll need to decide whether to enforce the icon header expectation. I'm inclined not to write any code for it, so if a client tries to put a long text header in an icon column they'll just get truncated text.